walletlibrary/src/test/java/com/microsoft/walletlibrary/mappings/issuance/DisplayContractMappingTest.kt

```diff
@@ -82,8 +82,8 @@ class DisplayContractMappingTest {
         assertThat(actualVerifiedIdStyle.backgroundColor).isEqualTo(expectedCardBackgroundColor)
         assertThat(actualVerifiedIdStyle.textColor).isEqualTo(expectedCardTextColor)
         assertThat(actualVerifiedIdStyle.logo).isNotNull
-        assertThat(actualVerifiedIdStyle.logo?.uri).isEqualTo(expectedLogoUri)
-        assertThat(actualVerifiedIdStyle.logo?.description).isEqualTo(expectedLogoDescription)
+/*        assertThat(actualVerifiedIdStyle.logo?.url).isEqualTo(expectedLogoUri)
```

Comment:
nit: commented out code

Reply:
Oh yes, I commented it out to test Uri. I will uncomment it.